client/stylesheets/_toCat.scss

```diff
@@ -3302,6 +3310,18 @@ On click the images will be displayed at normal size to complete the effect
     float: right;
 } 
 
+.listingItemLocationString {
+    margin-top: 18px;
+    padding: 10px 0px 10px 0px;
+    background-color: #f8504b;
+    border-radius: 4px;
+}
+
+.listingItemLocationString p {
+    color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces